src/layout/section/baseSection/baseSection.tsx

```diff
@@ -14,6 +14,7 @@ export interface BaseSectionProps {
   readonly role?: string
   readonly children: JSX.Element | string | React.ReactNode
   readonly contentSize?: SectionContentSize
+  readonly normalizedChildren?: boolean
```

Comment:
for the API naming, I think we should avoid technical terms and be more general like `nospacing`